process_onestop_sr_report/preprocessing.py

```diff
@@ -824,4 +839,19 @@ def validate_spacy_model(spacy_model_name: str) -> None:
 
 if __name__ == "__main__":
     cfg = ArgsParser().parse_args()
+    cfg.hf_access_token = "" # Add your huggingface access token here
+    cfg.data_path = Path("/data/home/shared/onestop/p_ia_reports")
+    cfg.save_path = Path("/data/home/shared/onestop/processed/ia_data_enriched_360_18042024.csv")
```

Comment:
should take from decoding repo, we have an up to date example that uses today's date (other wise people don't change it)